deploy/scripts/src/accounts/test.ts

```diff
@@ -716,3 +777,132 @@ export function deleteAccount (): void {
+      }), {
+      'is status 200': r => r.status === 200,
+      'verify page content': r => (r.body as string).includes('Enter your password')
+    }))
+
+  sleep(Math.random() * 3)
+
+  let acceptNewTerms = false
+  switch (userData.mfaOption) {
+    case 'AUTH_APP': {
+      res = group('B05_ValidateUser_04_AuthMFA_EnterPassword POST', () =>
+        timeRequest(() =>
+          res.submitForm({
+            fields: { password: credentials.currPassword },
+            params: { tags: { name: 'B05_ValidateUser_04_AuthMFA_EnterPassword' } }
+          }), {
+          'is status 200': r => r.status === 200,
+          'verify page content': r => (r.body as string).includes('Enter the 6 digit security code shown in your authenticator app')
+        }))
+
+      sleep(Math.random() * 3)
+
+      const totp = new TOTP(credentials.authAppKey)
+      res = group('B05_ValidateUser_05_AuthMFA_EnterTOTP POST', () =>
+        timeRequest(() => {
+          const response = res.submitForm({
+            fields: { code: totp.generateTOTP() },
+            params: { tags: { name: 'B05_ValidateUser_05_AuthMFA_EnterTOTP' } }
+          })
+          acceptNewTerms = (response.body as string).includes('terms of use update')
+          return response
+        }, {
+          'is status 200': r => r.status === 200,
+          'verify page content': r => acceptNewTerms || (r.body as string).includes('Services you can use with GOV.UK One Login')
+        }))
+      break
+    }
+    case 'SMS': {
+      res = group('B05_ValidateUser_06_SMSMFA_EnterPassword POST', () =>
+        timeRequest(() =>
+          res.submitForm({
+            fields: { password: credentials.currPassword },
+            params: { tags: { name: 'B05_ValidateUser_06_SMSMFA_EnterPassword' } }
+          }), {
+          'is status 200': r => r.status === 200,
+          'verify page content': r => (r.body as string).includes('Check your phone')
+        }))
+
+      sleep(1)
+
+      res = group('B05_ValidateUser_07_SMSMFA_EnterOTP POST', () =>
+        timeRequest(() => res.submitForm({
+          fields: { code: credentials.fixedPhoneOTP },
+          params: { tags: { name: 'B05_ValidateUser_07_SMSMFA_EnterOTP' } }
+        }),
+        {
+          'is status 200': r => r.status === 200,
+          'verify page content': r => (r.body as string).includes('Services you can use with GOV.UK One Login')
+        }))
+      break
+    }
+  }
+
+  if (acceptNewTerms) {
+    res = group('B05_ValidateUser_08_AcceptTermsConditions POST', () =>
+      timeRequest(() =>
+        res.submitForm({
+          fields: { termsAndConditionsResult: 'accept' },
+          params: { tags: { name: 'B05_ValidateUser_08_AcceptTermsConditions' } }
+        }), {
+        'is status 200': r => r.status === 200,
+        'verify page content': r => (r.body as string).includes('Services you can use with GOV.UK One Login')
+      }))
+  }
+
+  sleep(Math.random() * 3)
```

Comment:
Would suggest changing this sleep from a random wait to one that tries to sync up requests so they are called at the same time, to try and replicate defect conditions.

For example, to wait till the end of the next 5 second interval you could do:
```suggestion
  sleep((5000 - (Date.now() % 5000))/1000)
```
or to just send on the second mark
```suggestion
  sleep((1000 - (Date.now() % 1000))/1000)
```